### PR TITLE
Moose kitten finding

### PR DESCRIPTION
**Pull request by hyper2snyper**
```SUMMARY: Content "Adds the possiblitiy to find an evil moose in the kitten finding game."```

#### Purpose of change
I wanted to do something in C++

### CODE REVIEW

File: src/iuse_software_kitten.cpp [hunk 1]
```diff
@@ -215,6 +215,7 @@ std::string robot_finds_kitten::getmessage( int idx )
         _( "It's 1000 secrets the government doesn't want you to know!" ),
         _( "The letters O and R." ),
         _( "A magical... magic thing." ),
+        _( "That is a moose, a thing of pure evil. you should \"RUN!\""),
```
Comment (Night-Pryanik) [inline src/iuse_software_kitten.cpp]:
```suggestion
        _( "That is a moose, a thing of pure evil. You should \"RUN!\""),
```

Comment (hyper2snyper) [reply src/iuse_software_kitten.cpp]: Thanks

**Comment by Little-119** [comment]
Make it spawn a moose on the player.

**Comment by hyper2snyper** [comment]
>Make it spawn a moose on the player.

I would have no clue how to even start with that.

**Pull request merged**